Cargo.toml

```diff
@@ -1,3 +1,22 @@
+[workspace]
+resolver = "2"
+members = [
+  "crates/cargo-platform",
+  "crates/cargo-test-macro",
+  "crates/cargo-test-support",
+  "crates/cargo-util",
+  "crates/crates-io",
+  "crates/credential/*",
+  "crates/home",
+  "crates/mdman",
+]
+exclude = [
+  "benches/",
+  "crates/resolver-tests",
+  "src/doc/", # linkchecker and semver-check
```

Comment:
We could put an empty `[workspace]` in these to stop the workspace search.

An alternative future possibility is if we allow `package.workspace = <bool>` to turn on/off workspace searching

Reply:
Unfortunately, `linkchecker` is downloaded during CI pipeline. We either dynamically change Cargo.toml during CI or specify it in this `exclude`.

Reply:
And `linkchecker` is part of another repos workspace and it is integrated in enough that it doesn't make sense to do another process for this, like `cargo install --git`

Not a blocker for this but I do wonder if we should change `linkchecker.sh` to be more resilient for where its being run